packages/backend/src/webmonetization/model.ts

```diff
@@ -0,0 +1,27 @@
+import { BaseModel } from '../shared/baseModel'
+
+export class WebMonetization extends BaseModel {
+  public static get modelPaths(): string[] {
+    return [__dirname]
+  }
+
+  public static get tableName(): string {
+    return 'webMonetization'
+  }
+
+  static get idColumn(): string {
+    return 'accountId'
+  }
+
+  public accountId!: string
+  public currentInvoiceId!: string
+
+  public $beforeInsert(): void {
```

Comment:
Instead of reimplementing here, `BaseModel`'s `$beforeInsert` could generalize to `this[this.idColumn] = this[this.idColumn] || uuid()`.

Reply:
Yeah that seems better than this 👍

Reply:
The issue you now run into is that the id field is defined on the base model but isn't used. 

I think what may be better here is to rather just have the primary key be called `id` and put a comment that it is the `accountId`